.github/workflows/pre-commit.yaml

```diff
@@ -15,6 +15,7 @@ on:
 jobs:
   pre-commit:
     runs-on: ubuntu-22.04
+    timeout-minutes: 60
```

Comment:
Pre-commit should never hit a 60 minute timeout. If adding a timeout, 5 or 15 minutes might be more appropriate. I would just remove timeouts from this workflow file though.